src/main/draft/tomcat.yaml

```diff
@@ -0,0 +1,457 @@
+    Type: AWS::EC2::Image::Id
+    Default: ami-0f3a43fbf2d3899f7
+
+  keyName:
+    Description: The Key Manager key to be added to the list of allowed keys.
+    Type: AWS::EC2::KeyPair::KeyName
+
+  authorizedKeys:
+    Description: A URL pointing to a list of authorized SSH keys to be appended as-is to ~/.ssh/authorized_keys.
+    Type: String
+    MinLength: 1
+
+Mappings:
+  Scripts:
+    Tomcat:
+      InstallTomcat: |
+        #!/bin/bash -xe
+        wget -q -O - "https://repo1.maven.org/maven2/org/apache/tomcat/tomcat/9.0.8/tomcat-9.0.8.tar.gz" | tar xvzf - -C /opt
+        mv /opt/apache-tomcat-9.0.8 /opt/tomcat
+        mv /opt/tomcat/conf /etc/tomcat
+        ln -s /etc/tomcat /opt/tomcat/conf
+        mv /opt/tomcat/logs /var/log/tomcat
+        ln -s /var/log/tomcat /opt/tomcat/logs
+        hostname ${NAME}-$(curl http://169.254.169.254/latest/meta-data/instance-id)
+
+      InstallLogback: |
+        #!/bin/bash -xe
+        wget -O /opt/tomcat/lib/slf4j-api-1.7.19.jar "https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.19/slf4j-api-1.7.19.jar"
+        wget -O /opt/tomcat/lib/jul-to-slf4j-1.7.19.jar "https://repo1.maven.org/maven2/org/slf4j/jul-to-slf4j/1.7.19/jul-to-slf4j-1.7.19.jar"
+        wget -O /opt/tomcat/lib/logback-core-1.1.6.jar "https://repo1.maven.org/maven2/ch/qos/logback/logback-core/1.1.6/logback-core-1.1.6.jar"
+        wget -O /opt/tomcat/lib/logback-access-1.1.6.jar "https://repo1.maven.org/maven2/ch/qos/logback/logback-access/1.1.6/logback-access-1.1.6.jar"
+        wget -O /opt/tomcat/lib/logback-classic-1.1.6.jar "https://repo1.maven.org/maven2/ch/qos/logback/logback-classic/1.1.6/logback-classic-1.1.6.jar"
+        wget -O /opt/tomcat/lib/logback-syslog4j-1.0.0.jar "https://repo1.maven.org/maven2/com/papertrailapp/logback-syslog4j/1.0.0/logback-syslog4j-1.0.0.jar"
+        wget -O /opt/tomcat/lib/syslog4j-0.9.46.jar "http://clojars.org/repo/org/syslog4j/syslog4j/0.9.46/syslog4j-0.9.46.jar"
+
+Resources:
+  securityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupName: !Sub ${AWS::StackName} Security Group
+      GroupDescription: !Sub ${AWS::StackName} Security Group
+      VpcId:
+        Fn::ImportValue:
+          !Sub "${vpcStackName}-vpc"
+      Tags:
+        - Key: Name
+          Value: !Sub ${AWS::StackName} Security Group
+
+  bastion22Ingress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !Ref securityGroup
+      IpProtocol: tcp
+      FromPort: 22
+      ToPort: 22
+      SourceSecurityGroupId:
+        Fn::ImportValue:
+          !Sub "${vpcStackName}-bastion-security-group"
+
+  cassandra9042Ingress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId:
+        Fn::ImportValue:
+          !Sub "${cassandraStackName}-security-group"
+      IpProtocol: tcp
+      FromPort: 9042
+      ToPort: 9042
+      SourceSecurityGroupId: !Ref securityGroup
+
+  cassandra9042Ingress:
```

Comment:
Rename to `9160`.

Reply:
Interesting that this was even possible. Cloudformation did not complain...